sematic/db/migrations/20230221004028_add_id_to_users.py

```diff
@@ -0,0 +1,44 @@
+# Standard Library
+import uuid
+
+# Third-party
+from sqlalchemy.sql import text
+
+# Sematic
+from sematic.db.db import db
+
+
+def up():
+    with db().get_engine().begin() as conn:
+        users = conn.execute(
```

Comment:
Would `INSERT INTO users_tmp SELECT columns FROM users` work?

https://www.w3schools.com/sql/sql_insert_into_select.asp

Reply:
The uuid needs to be generated in Python.